types/node/test/fs.ts

```diff
@@ -374,7 +374,7 @@ async function testPromisify() {
 }
 
 async () => {
-    const handle: FileHandle = await openAsync('test', 'r');
```

Comment:
That's not a good test, please add a test case rather than altering an existing line.